### ISSUE TITLE
'TexEnv' object has no attribute 'everything'

### ISSUE THREAD

**Issue by jackassplus** [alvinwan/TexSoup#49]
In trying to run your Simple Conversion exampe, I get: 

line 64, in <module>
    tex_dict = {"latex": {"contents": to_dictionary(TexSoup.TexSoup(tex_text).expr.everything)}}
AttributeError: 'TexEnv' object has no attribute 'everything'

on TexSoup 0.1.3 from pip

**Comment by SimonMaenaut** [comment]
This example requires features implemented in #44 , which have not yet been pushed to pip. You can pull from [master](/alvinwan/TexSoup/tree/master) to receive the latest changes.

**Comment by alvinwan** [comment]
Thanks for reporting @jackassplus, and also many thanks for the contributions/attending to issues here @SimonMaenaut. Just pushed 0.1.4, so `everything` is now available

```
>>> list(TexSoup("\item ").expr.everything)
[TexCmd('item')]
```
